### DISCUSSION TITLE
Kibana Log persistence, restore deleted indices

### DISCUSSION THREAD

**Discussion by ccclever2** [telekom-security/tpotce#1250] [Q&A] [answered]
Hi,

we deployed T-Pot for around 30 days on multiple servers. As the wiki says, we increased the log persistence in `/opt/tpot/etc/logrotate/logrotate.conf` by increasing `rotate` to 60. However, data still has been removed from Kibana and we only see data from the last 30 days. Unfortunately, we forgot to increase the Index Lifecycle Policy and it is still at 30 days. 
The data seems to be still persistent in `/data`, is there anything we can do to get the data back in Kibana?

**Comment by t3chn0m4g3** [comment, marked as answer]
Not that I know of.

**Reply by ccclever2** [reply]
Ok, thanks anyways!
